libtransmission/session-id.h

```diff
@@ -51,8 +51,8 @@ class tr_session_id
 
     current_time_func_t const get_current_time_;
 
-    mutable session_id_t current_value_;
-    mutable session_id_t previous_value_;
+    mutable session_id_t current_value_{};
+    mutable session_id_t previous_value_{};
```

Comment:
_(nit)_ suggest `= {};`  AFAIK the code generated will be equivalent, but `= {};` is used all over libtransmission, so let's use it here too for consistency.

```suggestion
    mutable session_id_t current_value_ = {};
    mutable session_id_t previous_value_ = {};
```